### PR TITLE
Support/add crk token

### PR DESCRIPTION
**Pull request by quimodotcom**
### 📝 Description

Add logo for Cracker Token - CRK to ledger. Listing of Cracker Token have been requested on 8.12.2022.

Contract address (ERC-20)
CRK - 0x562Df594BFD9C8DDC894571bBB211DaC7888eaF6

### ❓ Context

- **Impacted projects**: `@ledgerhq/ledger-live`

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes**

### CODE REVIEW

**Comment by quimodotcom** [comment]
pls get this done asap!!!

File: .changeset/cuddly-fans-pump.md
Comment (quimodotcom) [reply .changeset/cuddly-fans-pump.md:2]: Ah lemme change that rq

**Comment by quimodotcom** [comment]
Changed from minor to patch. Awaiting review

**Review by mcayuelas-ledger — APPROVED** [review]

**Review by mcayuelas-ledger — APPROVED** [review]

**Comment by quimodotcom** [comment]
Apparently need a maintainer to approve the workflows due to me updating pr.

**Comment by mcayuelas-ledger** [comment]
> Apparently need a maintainer to approve the workflows due to me updating pr.

Yes done :)

**Comment by quimodotcom** [comment]
> > Apparently need a maintainer to approve the workflows due to me updating pr.
> 
> Yes done :)

ah thanks. the ones that have failed. is that ok or meant to happen?

**Comment by mcayuelas-ledger** [comment]
Seems that the SVG isn't valid with viewport

**Comment by quimodotcom** [comment]
> Seems that the SVG isn't valid with viewport

will fix asap.

**Comment by quimodotcom** [comment]
fixed

**Review by mcayuelas-ledger — APPROVED** [review]

**Comment by quimodotcom** [comment]
omg. ok lemme see what the problem with the svg is now

**Review by mcayuelas-ledger — APPROVED** [review]

**Comment by quimodotcom** [comment]
need workflow approval due to me updating to latest commit

**Review by mcayuelas-ledger — APPROVED** [review]

**Comment by quimodotcom** [comment]
Can you pls tell me wtf is going on??

**Comment by adrienlacombe-ledger** [comment]
> Can you pls tell me wtf is going on??

@quimodotcom some tests are failing, can you please rebase? thank you

**Comment by quimodotcom** [comment]
think i have rebased correctly. re-requesting approval

**Review by mcayuelas-ledger — APPROVED** [review]

**Comment by quimodotcom** [comment]
@mcayuelas-ledger do you know why that one test keeps failing?

**Pull request merged**
